compiler/rustc_feature/src/builtin_attrs.rs

```diff
@@ -421,6 +421,7 @@ pub const BUILTIN_ATTRIBUTES: &[BuiltinAttribute] = &[
         "the `#[profiler_runtime]` attribute is used to identify the `profiler_builtins` crate \
         which contains the profiler runtime and will never be stable",
     ),
+    gated!(rustc_main, AssumedUsed, template!(List: "path"), experimental!(rustc_main)),
```

Comment:
If this attribute is not going to be stabilized, then it doesn't need a separate feature and a tracking issue.
It can use `feature(rustc_attrs)` instead, see the attributes defined with `rustc_attr!` below.